packages/grid/ansible/roles/containers/tasks/hagrid.yml

```diff
@@ -8,3 +8,8 @@
   shell: "runuser -l {{ om_user }} -c 'pip install -e {{ syft_dir }}/packages/hagrid'"
   become: yes
   when: vagrant is defined and install == "true"
+
+- name: Install HAGrid
```

Comment:
This is identical to the rule above and its also the wrong rule as this one will only run if `vagrant` is defined which we use in local Vagrant vm mode.

Reply:
@madhavajay that was a weird text override.. was meant to be a click upgrade task.